Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  AddFeedbackContentViewController.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class AddFeedbackContentViewController: BaseViewController {
+    
+    // MARK: - property
+
+    private let backButton = BackButton()
+    private let closeButton = CloseButton()
+    private let addFeedbackTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.addFeedbackContentViewControllerTitleLabel
+        label.textColor = .black100
+        label.font = .title
+        return label
+    }()
+    private let feedbackTypeLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.addFeedbackContentViewControllerFeedbackTypeLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private let feedbackTypeButtonView = FeedbackTypeButtonView()
+    
+    // MARK: - lifecycle
+    
+    override func render() {
+        view.addSubview(addFeedbackTitleLabel)
+        addFeedbackTitleLabel.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide).offset(SizeLiteral.topPadding)
+            $0.leading.equalToSuperview().offset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(feedbackTypeLabel)
+        feedbackTypeLabel.snp.makeConstraints {
+            $0.top.equalTo(addFeedbackTitleLabel.snp.bottom).offset(40)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(feedbackTypeButtonView)
+        feedbackTypeButtonView.snp.makeConstraints {
+            $0.top.equalTo(feedbackTypeLabel.snp.bottom).offset(10)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.centerX.equalToSuperview()
```

Comment:
p2;

```suggestion
            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
```

전체 크기를 leading.trailing으로 레이아웃을 잡고 있어서 centeX는 지워도 될 것 같습니다 !
중복으로 레이아웃을 잡는 것 같아요 ^_______^

Reply:
[68e685d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/68e685de40d10b3a26a9ce71d4f906e12b8adc8c)
오호라 꼼꼼하게 봐주셔서 감사해요 ! 수정해서 반영해두었습니다 :)